src/simcore_service_deployment_agent/auto_deploy_task.py

```diff
@@ -421,6 +429,10 @@ async def auto_deploy(app: web.Application):
     except CancelledError:
         app["state"][TASK_NAME] = State.STOPPED
         return
+    except TagSyncErrorException:
+        log.exception(
```

Comment:
if this is expected, using ```log.exception``` will print the stack trace. Maybe that is not what you want? maybe this should be just a warning?

Reply:
thanks did not realize this, changing it to log.error